src/lib/i18n/dictionaries/dictionary.json

```
@@ -5506,5 +5506,12 @@
   "whatsOn.april28th": "April 28th",
   "whatsOn.ticketPaused": "The Bumpkins are on holidays - No more timeshards from chores & bounties.",
   "whatsOn.mysterySeasonText2": "No more timeshards from deliveries.",
-  "whatsOn.mysterySeasonText3": "No more ancient clocks while digging."
+  "whatsOn.mysterySeasonText3": "No more ancient clocks while digging.",
+  "ronin.nft.welcome": "Welcome Ronin Player!",
+  "ronin.nft.description": "Your {{nftName}} comes with exclusive perks!",
+  "ronin.nft.perkOne": "VIP Access",
+  "ronin.nft.perkTwo": "50% Faster Cooking",
```

```suggestion
  "ronin.nft.perkTwo": "2x Faster Cooking",
```
a text clarification here